keps/sig-node/compute-device-assignment.md

```diff
@@ -57,10 +59,15 @@ In this document we will discuss the motivation and code changes required for in
 
 ![device monitoring architecture](https://user-images.githubusercontent.com/3262098/43926483-44331496-9bdf-11e8-82a0-14b47583b103.png)
 
+### Device aware CNI plugin
+After this interface has been introduced it was used by CNI plugins like [kuryr-kubernetes](https://review.opendev.org/#/c/651580/) in couple with [intel-sriov-device-plugin](https://github.com/intel/sriov-network-device-plugin) to correctly define which devices were assigned to the pod.
+
+### Topology aware scheduling
+This interface can be used to collect allocated resources with information about the NUMA topology of the worker node. This information can then be used in NUMA aware scheduling.
```

Comment:
How come the information about already allocated resources can be used for future scheduling?
Device IDs are dynamic objects, device plugins can add/remove instances of the devices on the fly.
Previous information about same device ID can't be re-used for future decisions with 100% certainty.

Reply:
We mostly interesting in resource and its numaid. Deviceid itself is not so interesting.

Reply:
> We mostly interesting in resource and its numaid. Deviceid itself is not so interesting.

What do you mean by that statement? numa_id can't be assigned to container. It is a property of other resource instance.

Reply:
we're using the quantity of resources per numa node, that quantity is calculated by number of element in device list. We don't use device id here.
> numa_id can't be assigned to container. It is a property of other resource instance.
Frankly saying I didn't find *container* word in the thread above, except yours. 
Yes it can't be, since linux kernel doesn't have container entity, not so clear why *container* is mentioned here.

Reply:
"Container" word comes from changes below in `ContainerDevices`. `Device` instance can be uniquely mapped to what is announced by device plugins. Device plugins announcing device instances with `TopologyInfo` filed. this field contains `repeated numa_id` per device instance.